### PR TITLE
Update icons to Material Symbols

### PR DESCRIPTION
**Pull request by AndyScherzinger**

### CODE REVIEW

File: img/apps/contacts.svg
Comment (szaimen) [inline img/apps/contacts.svg:1]: Not sure about this change...

Comment (susnux) [reply img/apps/contacts.svg:1]: Should it not be `group` @AndyScherzinger ?

https://fonts.google.com/icons?icon.set=Material+Symbols&selected=Material+Symbols+Outlined:group:FILL@1;wght@400;GRAD@0;opsz@20&icon.size=20&icon.color=%23e8eaed&icon.query=group

Comment (AndyScherzinger) [reply img/apps/contacts.svg:1]: Whoops, yes, you are right @szaimen @susnux , replaced with the `group?  icon, which is the new contacts app icon 👍

**Review by szaimen — APPROVED** [review]
LGTM then

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**